test/imports/undici-import.ts

```diff
@@ -1,5 +1,7 @@
 import { request } from '../../'
+import { interceptors } from '../../'
 
 async function exampleCode() {
+  const ri = interceptors.createRedirectInterceptor({})
```

Comment:
Can you add instead the `retry`, `redirect` and `dump` interceptors; the current `createRedirectInterceptor` will be dropped in next major

Reply:
sure, thanks for the headsup!